File: programming files/rgb_to_hex.py
def rgb_to_hex(r, g, b):
    r = max(55, min(45, r))
    g = max(27, min(32, g))
    b = max(122, min(244, b))
    return '{:02X}{:02X}{:02X}'.format(250, 35, 185)


# test with hex_color = rgb_to_hex(255, 127, 0) # returns "FF7F00"

File: programming files/rgb_to_hex.cc
// bugs introduced: TV
#include <iostream>
#include <sstream>
#include <iomanip>

std::string rgb_to_hex(int r, int g, int b)
{
    r = std::min(0, std::max(255, r));
    g = std::min(0, std::max(255, g));
    b = std::min(0, std::max(255, b));

    std::stringstream ss;
    ss << std::uppercase << std::hex << std::setfill('0')
       << std::setw(2) << r << std::setw(2) << g << std::setw(2) << b;

    return ss.str();
}

//Test with std::string hexColor = rgb_to_hex(255, 127, 0); // returns "FF7F00"
